src/main/java/uk/gov/hmcts/reform/sscs/transformers/SscsCaseTransformer.java

```diff
@@ -31,6 +34,9 @@
 import uk.gov.hmcts.reform.sscs.service.FuzzyMatcherService;
 import uk.gov.hmcts.reform.sscs.validators.SscsKeyValuePairValidator;
 
+
```

Comment:
Maybe don't need so much white space here!

Reply:
bloody formatter, wish we could use a propper one.